reV/supply_curve/aggregation.py

```diff
@@ -1,64 +1,39 @@
 # -*- coding: utf-8 -*-
-"""reV supply curve aggregation framework.
-
-Created on Fri Jun 21 13:24:31 2019
-
-@author: gbuster
 """
+reV aggregation framework.
+"""
+from abc import ABC, abstractmethod
 from concurrent.futures import as_completed
-import os
+import logging
 import h5py
 import numpy as np
+import os
 import pandas as pd
-from scipy.spatial import cKDTree
-from warnings import warn
-import logging
 
 from reV.handlers.outputs import Outputs
-from reV.handlers.exclusions import ExclusionLayers
-from reV.supply_curve.exclusions import ExclusionMaskFromDict, FrictionMask
-from reV.supply_curve.points import SupplyCurveExtent
-from reV.supply_curve.point_summary import SupplyCurvePointSummary
+from reV.supply_curve.exclusions import ExclusionMaskFromDict
+from reV.supply_curve.points import (SupplyCurveExtent,
+                                     AggregationSupplyCurvePoint)
 from reV.utilities.execution import SpawnProcessPool
 from reV.utilities.exceptions import (EmptySupplyCurvePointError,
-                                      OutputWarning, FileInputError,
-                                      InputWarning, SupplyCurveInputError)
+                                      FileInputError)
 
 
 logger = logging.getLogger(__name__)
 
 
-class AggFileHandler:
+class AbstractAggFileHandler(ABC):
```

Comment:
The abstract vs. concrete versions of this seem redundant. Maybe differentiate the docstrings to specify use cases?

Reply:
the main difference is we added the (Resource) h5 file handler